### DISCUSSION TITLE
1.16.4哪里来的“Forge 36 or above”啊

### DISCUSSION THREAD

**Discussion by kong-que-dnf123** [Meloong-Git/PCL#2898] [提问] [answered]
这forge最高就35.1.37 哪来的36 or above啊
![image](https://github.com/Hex-Dragon/PCL2/assets/120994373/a1e9534e-765c-4b8c-a4ac-dba3312b086d)
![image](https://github.com/Hex-Dragon/PCL2/assets/120994373/ac18fccc-213b-4130-bec6-7ea2c0d3b571)
![image](https://github.com/Hex-Dragon/PCL2/assets/120994373/fb17c740-7e00-41f4-8eaf-f594ce044406)

错误报告文件：
[错误报告-2023-11-25_16.03.09.zip](https://github.com/Hex-Dragon/PCL2/files/13463332/-2023-11-25_16.03.09.zip)

**Comment by MoYuan-CN** [comment]
有可能这个 Mod 在 1.16.4 的 MC 客户端下不可用
我转到那边仓库问问（见 MCTeamPotato/MultiOfflineFix#3）

**Comment by Pigeon0v0** [comment, marked as answer]
所以你应该用 1.16.5 ...

**Reply by MoYuan-CN** [reply]
或许是个好主意

**Comment by PanboYuisshuaige** [comment]
我也遇到同样问题了

**Comment by Silverteal** [comment]
题外话：1.16.4和1.16.5的游戏内容几乎完全相同，而1.16.5发布的目的是修复一个1.16.4发布时没有发现的重要bug
